packages/storybook/stories/va-accordion.stories.jsx

```diff
@@ -10,7 +10,8 @@ const accordionDocs = getWebComponentDocs('va-accordion');
 const accordionItem = getWebComponentDocs('va-accordion-item');
 
 export default {
-  title: 'Components/va-accordion',
+  title: 'Components/Accordion/va-accordion',
+  id: 'Components/va-accordion',
```

Comment:
How is this ID being used? I looked at the [storybook documentation](https://storybook.js.org/docs/react/api/csf) and it wasn't obvious.

Reply:
This id is used as a [permalink](https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalink-to-stories) to prevent having to update links if the component structure changes in Storybook.